src/java.desktop/macosx/native/libawt_lwawt/awt/JavaComponentAccessibility.m

```diff
@@ -1960,6 +1972,27 @@ - (nullable id)accessibilityParent
 
 @end
 
+
+/*
+ * Implementation of the NSAccessibilityButton protocol
+ */
+@implementation ButtonAccessibility
```

Comment:
Do not we need to remove the old `perform()`?

Reply:
Not yet - as i said i'm not going to delete the old code yet si it can be easily reverted for testing and comparison reasons.